compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/nodes/GraphState.java

```diff
@@ -61,23 +61,22 @@ public final class GraphState {
                     StageFlag.SAFEPOINTS_INSERTION,
                     StageFlag.GUARD_LOWERING,
                     StageFlag.MID_TIER_LOWERING,
-                    StageFlag.FSA,
-                    StageFlag.BARRIER_ADDITION);
+                    StageFlag.FSA);
     private static final EnumSet<StageFlag> LOW_TIER_MANDATORY_STAGES = EnumSet.of(
                     StageFlag.LOW_TIER_LOWERING,
                     StageFlag.EXPAND_LOGIC,
                     StageFlag.ADDRESS_LOWERING,
                     StageFlag.REMOVE_OPAQUE_VALUES,
-                    StageFlag.FINAL_SCHEDULE);
+                    StageFlag.FINAL_SCHEDULE,
```

Comment:
Some unit tests were failing because this list hadn't been updated after our changes.